src/init-ceph.in

```diff
@@ -740,7 +740,27 @@ if [ "$command" = "stop" -o "$command" = "onestop" ]; then
     what="$new_order"
 fi
 
+# Check if the monitors are up before starting any mds
```

Comment:
Expand on the comment on my we only need this on Standard installs vs AIOs

Reply:
done